.github/trivy/policy-backend-fossa.json

```diff
@@ -0,0 +1,3576 @@
+{
```

Comment:
I recommend to change the name to `policy-backend-fossa-OLD.json`

Reply:
What do you think of the very explicit name `policy-backend-fossa-reference-2024-06-11.json` as pushed [here](https://github.com/SDA-SE/sda-spring-boot-commons/pull/735/commits/8354ee601dc1d355ad4aafb6a34f6a558f0a5d4b)?

Reply:
If you name some file _old_ or _new_, you end up with _new-new_, _newer_, _old-but-not-so-old_ and so on 😉